src/api/todos.ts

```diff
@@ -1,10 +1,16 @@
 import { Todo } from '../types/Todo';
 import { client } from '../utils/fetchClient';
 
-export const USER_ID = 0;
+export const USER_ID = 1441;
 
 export const getTodos = () => {
   return client.get<Todo[]>(`/todos?userId=${USER_ID}`);
 };
 
-// Add more methods here
+export const getActive = () => {
+  return getTodos().then(todos => todos.filter(item => !item.completed));
+};
+
+export const getCompleted = () => {
+  return getTodos().then(todos => todos.filter(item => item.completed));
+};
```

Comment:
You don't need to send 2 requests to get todos with different completed status, you can filter them in app component instead.